Docs: How To Change The Base Branch

dericksozo opened the pull request:
(docs): Adding a section on how to change the base branch for both Sync with GitHub and Sync With Bitbucket.

dericksozo commented:
@GreenMachine01, I added this section to both Sync With Bitbucket and Sync With GitHub. What are your thoughts? Eventually we might have to put this somewhere else after we add AWS CodeCommit.

GreenMachine01 commented:
@dericksozo 
there is an important piece that is missing (which wasn't in the ticket - so you couldn't know)
this capability is only available to Pro and Enterprise users.
as such I would find ways to highlight it as part of the benefits of upgrading.

second, I think it makes more sense to create a dedicated page to cover this feature and link to it from any other relevant page versus holding copies of the same instructions on separate pages. also, users might not be interested in this feature and this will allow them to quickly skip over it if it is not relevant to them especially if they are free plan users.

dericksozo commented:
Hey @GreenMachine01, I just updated the structure. Let me know what you think!

GreenMachine01 commented on docs/how-to/change-base-branch.md:
```diff
@@ -0,0 +1,39 @@
+For example, you may want pull requests to merge into a development branch first before your main branch.
+
+:::info
+This feature is only available for **Enterprise Plan** users.
```
@dericksozo this feature is also available in the Pro plan.

GreenMachine01 commented on docs/how-to/change-base-branch.md:
```diff
@@ -0,0 +1,39 @@
+1. Go to your _project's_ Git Settings page.
+2. You'll see a new **Base Branch** text field.  
+3. Enter the name of the branch you want Amplication to merge changes into.
+4. Click **Save Changes**.
```
@dericksozo there is no "save changes" button for the user to click
the config is saved when the user exits the input field

GreenMachine01 commented on docs/how-to/change-base-branch.md:
```diff
@@ -0,0 +1,39 @@
+3. [Re-select](#select-your-repository) your GitHub repository.
+4. You'll see a new **Base Branch** text field.
+5. Enter the name of the branch you want Amplication to merge changes into.
+6. Click **Save Changes**.
```
@dericksozo same as above.

GreenMachine01 commented on docs/getting-started/sync-with-bitbucket.md:
```diff
@@ -67,3 +67,4 @@ And there you have it – a new pull request ready for review.
 
 ![bitbucket-pull-request](./assets/sync-with-git-provider/bitbucket-pull-request.png)
 
+Amplication creates pull requests from the `amplication` branch into your repository's default base branch. As an Enterprise plan user, you can [change the base branch for these pull requests](/how-to/change-base-branch-for-pull-requests).
```
@dericksozo as mentioned in other comments this is a Pro feature.

GreenMachine01 commented on docs/getting-started/sync-with-git-diffrences-between-plans.md:
```diff
@@ -25,7 +25,8 @@ When synchronizing with git provider using the Pro or Enterprise Plan, the follo
 ### Sync with Git provider (Enterprise Plans)
-- support for Bitbucket
+- Support for [Bitbucket](/sync-with-bitbucket)
+- [Change the default base branch for pull requests](/how-to/change-base-branch-for-pull-requests)
```
@dericksozo as mentioned in other comments this is a Pro feature.

GreenMachine01 commented on docs/getting-started/sync-with-github.md:
```diff
@@ -155,7 +155,11 @@ Let's review what's happening here in its entirety.
+Users can merge Amplication-made changes by merging pull requests created by Amplication from the `amplication` branch to the user repository's base branch.
+
+:::tip
+Enterprise plan users can [change the base branch](/how-to/change-base-branch-for-pull-requests) that Amplication uses for pull requests.
```
@dericksozo as mentioned in other comments this is a Pro feature.

dericksozo commented:
Hey @GreenMachine01, I just implemented all your feedback. I changed the title of the plan differences page to `Git Sync Plan Differences`. Let me know if that's alright.
